apps/frontend/core-dapp/src/features/position/PositionsNext.tsx

```diff
@@ -0,0 +1,213 @@
+import { Box, Button, Flex, Grid, Icon, spacingIncrement, Subtitle, Typography } from 'prepo-ui'
+import { observer } from 'mobx-react-lite'
+import { Trans } from '@lingui/macro'
+import styled from 'styled-components'
+import React from 'react'
+import { Tooltip } from 'antd'
+import { useRouter } from 'next/router'
+import ClosePositionSummary from './ClosePositionSummary'
+import PositionLabel from './PositionLabel'
+import { useRootStore } from '../../context/RootStoreProvider'
+import { Routes } from '../../lib/routes'
+import { RecordSkeleton } from '../portfolio/Record'
+import { PositionEntity } from '../../stores/entities/Position.entity'
+import { numberFormatter } from '../../utils/numberFormatter'
+import Percent from '../../components/Percent'
+
+const { toUsd } = numberFormatter
+
+const PositionName = styled.p`
+  color: ${({ theme }): string => theme.color.secondary};
+  font-size: ${({ theme }): string => theme.fontSize.md};
+  font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
+  margin-bottom: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`
+
+const PositionValue = styled.p`
+  color: ${({ theme }): string => theme.color.neutral1};
+  margin: 0;
+`
+
+const StyledSubtitle = styled(Subtitle)`
+  font-size: ${({ theme }): string => theme.fontSize.sm};
+  font-weight: ${({ theme }): number => theme.fontWeight.medium};
+`
+
+const StyledPositionLabel = styled(PositionLabel).attrs({ withBackground: true })`
+  font-size: ${({ theme }): string => theme.fontSize.sm};
+  font-weight: ${({ theme }): number => theme.fontWeight.regular};
+`
+
+const PlusButton = styled.button`
+  align-items: center;
+  appearance: none;
+  background: ${({ theme }): string => theme.color.accentPurple};
+  border: none;
+  border-radius: 0.375rem;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  padding: 2px;
+  transition: opacity 200ms;
+
+  :hover {
+    opacity: 0.8;
+  }
+`
+
+type PositionProps = {
+  direction: PositionEntity['direction']
+  growthPercentage: PositionEntity['positionGrowthPercentage']
+  iconName: PositionEntity['market']['iconName']
+  name: PositionEntity['market']['name']
+  onDecrementPosition: () => void
+  onIncrementPosition: () => void
```

Comment:
Can omit "Position" because this is Prop is for Position so it's implied

Also use verb for better naming convention (e.g. html's onChange, onDrag, onFocus)

```suggestion
  onDecrease: () => void
  onIncrease: () => void
```